nestkernel/node_manager.h

```diff
@@ -327,6 +349,24 @@ NodeManager::any_node_uses_wfr() const
   return any_node_uses_wfr_;
 }
 
+inline SparseNodeArray::const_iterator
+NodeManager::local_nodes_begin() const
+{
+  return local_nodes_.begin();
+}
+
+inline SparseNodeArray::const_iterator
+NodeManager::local_nodes_end() const
+{
+  return local_nodes_.begin();
```

Comment:
@suku248 Thank you for catching this! Nasty copy paste error, and rather unfortunate that it it not trigger any test. I will try to create tests that detect this.